docs-src/concepts/what-is-a-search-attribute.md

```diff
@@ -29,23 +29,28 @@ A Temporal Cluster has a set of default Search Attributes already available.
-| ExecutionStatus       | Keyword  | The current state of the Workflow Execution.                                                                                                                                 |
-| StartTime             | Datetime | The time at which the Workflow Execution started.                                                                                                                            |
-| CloseTime             | Datetime | The time at which the Workflow Execution completed.                                                                                                                          |
-| ExecutionTime         | Datetime | Same as StartTime for the most cases but different for cron Workflows and retried Workflows. For them it is the time at which the Workflow Execution actually begin running. |
-| RunId                 | Keyword  | Identifies the current Workflow Execution Run.                                                                                                                               |
-| ExecutionDuration     | Int      | The time needed to run the Workflow Execution. Available only for closed Workflows.                                                                                          |
-| HistoryLength         | Int      | The number of events in the history of Workflow Execution. Available only for closed Workflows.                                                                              |
-| StateTransitionCount  | Int      | The number of times that Workflow Execution has persisted its state. Available only for closed Workflows.                                                                    |
-| TaskQueue             | Keyword  | Task Queue used by Workflow Execution.                                                                                                                                       |
-| TemporalChangeVersion | Keyword  | If Workflow versioning is enabled, list of change/version pairs will be stored here.                                                                                         |
-| BinaryChecksums       | Keyword  | List of binary Ids of Workers that run the Workflow Execution.                                                                                                               |
-| BatcherNamespace      | Keyword  | Used by internal batcher to indicate the Namespace where batch operation was applied to.                                                                                     |
-| BatcherUser           | Keyword  | Used by internal batcher to indicate the user who started the batch operation.                                                                                               |
+| NAME                       | TYPE     | DEFINITION                                                                                                                                                                   |
```

Comment:
I would group these in some logical sections. May be even add line between them.

Section 1 (most used): WorkflowId, RundId, WorkflowType, ExecutionStatus, BatcherUser
Section 2 (times): StartTime, ExecutionTime, CloseTime, ExecutionDuration
Section 3 (sizes): StateTransitionCount, HistoryLength, HistorySizeBytes
Section 4 (versioning): TaskQueue, TemporalChangeVersion, BinaryChecksums, BuildIds
Section 5 (scheduler): TemporalScheduledStartTime, TemporalScheduledById, TemporalSchedulePaused


or at least sort them in this order.